trio/_core/_io_windows.py

```diff
@@ -350,19 +532,17 @@ def abort(raise_cancel_):
                     # does, we'll assume the handle wasn't registered.
```

Comment:
In this comment block there's a reference to the IOCP thread which needs to be updated